package.json

```
@@ -13,9 +13,9 @@
   "homepage": "https://github.com/Brightspace/user-tile",
   "name": "user-tile",
   "scripts": {
-    "lint": "npm run lint:wc && npm run lint:js",
+    "lint": "npm run lint:js",
     "lint:js": "eslint . --ext .js,.html test/**/*.js test/**/*.html demo/**/*.js demo/**/*.html",
-    "lint:wc": "polymer lint -i \"./*\" \"test\" \"demo\"",
+    "lint:lit-analyzer": "lit-analyzer *.js demo/**/* --strict --rules.no-unknown-tag-name off --rules.no-unknown-event error --maxWarnings 0",
```

```suggestion
    "lint:lit-analyzer": "lit-analyzer *.js demo/**/* test/**/* --strict --rules.no-unknown-tag-name off --rules.no-unknown-event error --maxWarnings 0",
```

My bad, I earlier offered an incomplete suggestion. Since the test fixtures aren't rendered via lit-html templates/via the LitElement `render()` method, it currently can't detect any issues in the tests yet, but if/when that's flipped over (not a blocker on this PR), it will report issues there also.